Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -31,22 +31,25 @@
 from Validation.RecoEgamma.photonMiniAODValidationSequence_cff import *
 from Validation.RecoEgamma.egammaValidationMiniAOD_cff import *
 
+prevalidationNoHLT = cms.Sequence( globalPrevalidation * metPreValidSeq * jetPreValidSeq )
 prevalidation = cms.Sequence( globalPrevalidation * hltassociation * metPreValidSeq * jetPreValidSeq )
 prevalidationLiteTracking = cms.Sequence( prevalidation )
 prevalidationLiteTracking.replace(globalPrevalidation,globalPrevalidationLiteTracking)
 prevalidationMiniAOD = cms.Sequence( genParticles1 * miniAODValidationSequence * photonMiniAODValidationSequence * egammaValidationMiniAOD)
 
 
-validation = cms.Sequence(cms.SequencePlaceholder("mix")
-                         +genvalid_all
-                         *globaldigisanalyze
-                         *globalhitsanalyze
-                         *globalrechitsanalyze
-                         *globalValidation
+validationNoHLT = cms.Sequence(cms.SequencePlaceholder("mix")
+                               +genvalid_all
+                               *globaldigisanalyze
+                               *globalhitsanalyze
+                               *globalrechitsanalyze
+                               *globalValidation)
+
+validation = cms.Sequence(validationNoHLT
                          *hltvalidation)
 
 _validation_fastsim = validation.copy()
-for _entry in [globaldigisanalyze,globalhitsanalyze,globalrechitsanalyze]:
+for _entry in [globaldigisanalyze,globalhitsanalyze,globalrechitsanalyze,hltvalidation]:
```

Comment:
If you do this, there is no need to define a separate `validationNoHLT` sequence above, right ?
Or do you have other use cases in mind ?